modules/manage/pages/iceberg/access-iceberg-topics.adoc

```diff
@@ -0,0 +1,103 @@
+= Access Topic Data in Iceberg Tables
```

Comment:
@kbatuigas the Access page the Query page are missing the admonition at the top of the page about needing an Enterprise license.

Reply:
Updated